server/tasks/stream_inference.py

```diff
@@ -71,7 +71,7 @@ async def run_streaming(
     task_time_limit=settings.task_time_limit_sec,
 )
 def streaming_parent_task(
-    self, agent: Agent, result_handler: ResultHandler, batch_size: int = 10
+    self, agent: Agent, result_handler: ResultHandler, batch_size: int = 1
```

Comment:
Any chance you've tested this on an FB with a reasonably large dataset? My only concern is that we create congestion in the DB from writing on each pred

Reply:
We are not calling LSE endpoint per record. See comment above

Reply:
To answer your question yes Ive tried with thousands of records on the LSE FB using this adala deployment.